public/templates/demo.html

```diff
@@ -18,7 +18,7 @@
     <link
       rel="shortcut icon"
       type="image/x-icon"
-      href="https://cdn.glitch.com/aa3eaa29-7183-4f33-b4e2-c3a30c833a25%2F31540120.jpg?v=1598133005255"
+      href="/public/assets/images/favicon.jpg"
```

Comment:
Could you also migrate the existing `png` images to the `jpg` format? Seems like `jpg` would result in faster page loads according to this article:

https://www.bluearcher.com/blog-item-jpg-vs-png-for-web#:~:text=The%20PNG%20file%20type%20holds,a%20difference%20for%20your%20audience.

Search the web for how to convert a png to a jpg.

Reply:
Okay so from the issue you filed (#92) you mention they have to be PNGs. So should we make everything PNG instead of these new ones being jpg?

Reply:
@j0n3lson No. The png files that I changed to jpg's work well and the images that I imported as jpg's (the files I added in my first commit) display correctly. It is only the specific service-icon files (1-3) that need to stay as pngs. I will delete the unused png image files and the temp/patientSignUp.html in a new commit.